samples/subsys/tracing/sample.yaml

```diff
@@ -10,11 +10,13 @@ tests:
   tracing.backends.sysview:
     platform_whitelist: nrf52840dk_nrf52840
     extra_configs:
+      - CONFIG_TRACING=y
       - CONFIG_SEGGER_SYSTEMVIEW=y
       - CONFIG_USE_SEGGER_RTT=y
```

Comment:
Does the `min_ram: 32` configuration need to be preserved from `samples/philosphers/sample.yaml` (removed in the other commit on this PR because the content resides elsewhere (here))?

Reply:
Dont think it is needed, those numbers are outdated in many cases.

Reply:
got it, thanks!